android/source/VideoLocker/src/org/edx/mobile/view/CourseChapterListFragment.java

```diff
@@ -203,6 +204,10 @@ public void onStart() {
     public void onStop() {
         super.onStop();
         isActivityStarted = false;
+        //We need to cancel the getLastAccessed task if the fragment is stopped
+        if(getLastAccessedTask!=null){
+            getLastAccessedTask.cancel(true);
```

Comment:
Yes @nasthagiri and @rohan-dhamal-clarice we will update all background tasks to cancel if the activity is stopped/paused. As Rohan suggested can we take those changes on a seperate branch.